metatron-app/metatron-discovery
# F#1825 map chart tooltip

### Description
1. When the spatial analysis is executed, the value information of the tooltip is displayed differently.
2. If choropleth is the default value, no value information is displayed in the tooltip

**Related Issue** :  #1825 

### How Has This Been Tested?
1. Go to mapchart
2. Click on spatial analysis
3. See error

![image](https://user-images.githubusercontent.com/42328560/55934295-1714b580-5c6b-11e9-90bb-f222fab013d4.png)


### Types of changes



### Checklist:
